suseRegisterInfo/suseRegister/info.py

```diff
@@ -49,6 +49,10 @@ def getRedHatLikeProducts():
         ret = parseReleaseInfo(release='/etc/centos-release')
         if ret:
             ret['name'] = "CentOS"
+    elif os.path.exists('/etc/system-release'):
+        ret = parseReleaseInfo(release='/etc/system-release')
+        if ret:
+            ret['name'] = "AmazonLinux"
```

Comment:
suseRegisterInfo is AFAIK only used for traditional clients. Not sure if you want to support it?